frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/useAutoLayout/useAutoLayout.ts

```diff
@@ -0,0 +1,28 @@
+import { useNodesInitialized, useReactFlow } from '@xyflow/react'
+import { useCallback, useEffect } from 'react'
+import { getElkLayout } from './getElkLayout'
+
+export const useAutoLayout = () => {
+  const nodesInitialized = useNodesInitialized()
```

Comment:
👀 https://reactflow.dev/api-reference/hooks/use-nodes-initialized

Reply:
I used this method because performing layout calculations before the initial display is complete can lead to undesired output results.